forecasters/animalia/R/production_forecaster.R

```diff
@@ -30,6 +30,12 @@
 #'   specified, defaults to the levels required by the COVID Forecast Hub.
 #' @param lambda vector of values to use for the regularization parameter
 #'   in quantile lasso
+#' @param norm_by_popn Should the response/features be normalized by population?
```

Comment:
rename to be consistent with the above version

Reply:
Renamed.